### ISSUE TITLE
Cutoff at the bottom of all "list-type" menus.

### ISSUE THREAD

**Issue by asterne-labs** [kzahel/jstorrent-legacy-app#24]
As the title says, I'm experiencing cutoff at the bottom of the page for all menus that act as a list, such as the "Files" menu, the "Peers" menu, and the "Swarm" menu when scrolled all the way to the bottom.
![screenshot 2014-03-24 at 6 54 20 pm](https://f.cloud.github.com/assets/7050266/2506488/b7abec9c-b3a7-11e3-9189-3e6a57591683.png)
![screenshot 2014-03-24 at 6 54 56 pm](https://f.cloud.github.com/assets/7050266/2506493/c3849582-b3a7-11e3-858e-fd8baeef5336.png)
![screenshot 2014-03-24 at 6 55 06 pm](https://f.cloud.github.com/assets/7050266/2506495/cfdb08ca-b3a7-11e3-91f1-a9f40ab905f4.png)

**Comment by asterne-labs** [comment]
Presumably this would also be applicable to the other list type menus, if they became populated enough, but I've yet to find a situation where they would be extensively populated like these three menus.

**Comment by kzahel** [comment]
Yup, I see it on my end too. I introduced this bug when I added the
colored/custom top menu bar, and it looks like the window height
calculation has gone off... looking into this.

**Comment by kzahel** [comment]
The legacy Chrome App is no longer maintained. I've rebuilt JSTorrent from scratch — the new version is ready at **[jstorrent.com](https://jstorrent.com)**

If you run into any issues, please leave feedback!
